### DISCUSSION TITLE
MCP(Model Context Protocol) is really cool for LLM applications!

### DISCUSSION THREAD

**Discussion by alanxmay** [open-webui/open-webui#7361] [Ideas]
LSP is really cool, lots of editor and language server implementations. [MCP](https://modelcontextprotocol.io/quickstart) just like LSP but for LLM applications.

Can't wait something happened with open-webui.

**Comment by punkpeye** [comment]
Technical details for those looking to implement it https://glama.ai/blog/2024-11-25-model-context-protocol-quickstart

**Comment by Xyz00777** [comment]
Isn't these something who also should be implemented in ollama? To enable also direct access?
